saleor/graphql/discount/mutations/sale_create.py

```diff
@@ -90,9 +89,9 @@ def clean_instance(cls, info, instance):
             raise ValidationError({"end_date": error})
 
     @classmethod
-    @traced_atomic_transaction()
     def perform_mutation(cls, _root, info, **data):
-        response = super().perform_mutation(_root, info, **data)
+        with traced_atomic_transaction():
```

Comment:
Is it expected that we reduced size of transaction here? Like previously we were sending `sale_created` with `transaction.on_commit` but now I see, that only perform_mutation is wrapped by transaction block